.github/pull_request_template.md

```diff
@@ -1,22 +1,32 @@
-<!--
-  Copy the task id from the bottom of the sidebar and paste it after FTRACK-
+<!--(
+  Copy the id and paste it to the appropriate CLICKUP-<id> / FTRACK-<id> /  SENTRY-<id> / ZENDESK-<id> link.
+  Please remember to remove the unused ones.
 -->
+Resolves : 
 
-Resolves FTRACK-
+* CLICKUP-
+* FTRACK-
+* SENTRY-
+* ZENDESK-
 
-- [ ] I have added automatic tests where applicable
-- [ ] The PR title is suitable as a release note
-- [ ] The PR contains a description of what has been changed
-- [ ] The description contains manual test instructions
+- [ ] I have added automatic tests where applicable.
+- [ ] The PR contains a description of what has been changed.
+- [ ] The description contains manual test instructions.
+- [ ] The PR contains update to the release notes.
+- [ ] The PR contains update to the documentation.
```

Comment:
In my mind, this list should be a todo list for the PR creator to make sure they have done all the necessary steps, and not a list that describes what has changed within the PR (since that is obvious from the file diffs). 

That is, when PRs are opened, this list should optimally be 100% checked every time. As such, I would rephrase the last point to:

```suggestion
- [ ] The PR contains updates to the release notes.
- [ ] I have verified that the documentation is still up to date.
```

Reply:
done now